flipper-plugin-realm/src/components/DataTable.tsx

```diff
@@ -99,20 +110,19 @@ PropertyType) => {
 
   const [rowExpansionProp, setRowExpansionProp] = useState({
     expandedRowRender: () => {
-      return;
+      return <></>;
```

Comment:
Does this work as well?
```suggestion
      return null;
```

Reply:
This would probably work if I also set a type definition for this field to be i.e. JS element | null, but otherwise later typescript will complain about functions not matching the type.